lib/messages.js

```diff
@@ -39,5 +39,26 @@ fns.newLocationMessage = (message, next) => {
             .catch(next);
     });
 };
+fns.noTests = (message, next) => {
+    Joi.validate(message.data, schema.locationMessage, (err, messageData) => {
+        if (err) {
+            return next(err);
+        }
+        messageData.message_type = 'location';
+
+        db.addNewMessage(messageData)
+            .then(dbResponse => {
+                next(null, {
+                    _id: dbResponse.insertedId
+                });
+            })
+            .catch(next);
+    });
+};
+fns.noTestsAgain = (message, next) => {
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'next' is defined but never used.